### PR TITLE
Feature/redux search instructors

### PR DESCRIPTION
**Pull request by JorgeEvangelista500**
## Description

Connected Search instructors  to redux and the back end, users can now search through existing instructors.

Collaborators:
[Allison Stewart](https://github.com/am-stewart)
## Loom Video

https://www.loom.com/share/7144eec3c20b4f7eb86d17115dc4f6d3

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by brandon-bee — APPROVED** [review]
Overall really clean and readable code. Left the one comment where I saw a button could be switched with the Ant Design component, but that was the only thing I saw. Nice work!

File: src/components/pages/LandingInstructor/SearchInstructors.js [hunk 1]
```diff
@@ -62,11 +73,22 @@ const SearchInstructors = () => {
                     <Option value="weekends">Weekends</Option>
                   </Select>
                 </Item>
-                <button className="il__top__formBtn" type="submit">
+                <button
```
[the remark this answers is not in the document]
Comment (JorgeEvangelista500) [reply src/components/pages/LandingInstructor/SearchInstructors.js:76]: That sounds like a good idea!, i kept the same design, i just added the on Click function

File: src/components/pages/LandingInstructor/SearchInstructors.js [hunk 2]
```diff
@@ -85,7 +85,6 @@ const SearchInstructors = props => {
             <div>
               {props.instructors.map(instructor => (
                 <div>
-                  {' '}
```
Comment (Berenika14) [inline src/components/pages/LandingInstructor/SearchInstructors.js]: Thanks for removing empty brackets and also the unneeded consoles.

**Review by Berenika14 — APPROVED** [review]
Thanks for removing empty brackets and also the unneeded consoles. Might need a Loom video in order to be merged. Overall Great Job Jorge the code looks clean and readable.

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This PR needs a Loom video before I can merge it! 😱 

Please add one as soon as you can!

**Review by aamay022 — APPROVED** [review]
Great work on implementing instructor reducer and actions to be able to view all instructors. Nice clean up on the console logs as well.

**Review by bugsbrog — APPROVED** [review]
Ooh, snazzy how you’re using Okta authentication in the SearchInstructors component! And nice work on making the button clickable! Ooh, love the dummy data (I think it's easier to work with actual data, whether it be dummy data or not 😆). I'd say get rid of the commented-out code in instructorActions.js (nice job on getting rid of "rogue console.logs"). Overall, GREAT work so far!

**Review by paulstgermain — APPROVED** [review]

**Pull request merged**